docs/community/release-checklist.md

```diff
@@ -118,8 +118,10 @@ Finally, let's make the new aks-engine release easy to install.
 Create a pull request to add the new release to [gofish][] through the [fish-food repository][gofish-food]. You will need to calculate the sha256 checksum for each of the .tar.gz archives:
 
 ```
-$ shasum -a 256 _dist/aks-engine-$TAG-darwin-amd64.tar.gz  # macOS example
-250c0b645ad22514f4af52393ebcda95f1f911032274801f17c295800741f75b  _dist/aks-engine-v0.32.3-darwin-amd64.tar.gz
+$ for tarball in darwin-amd64 linux-amd64 windows-amd64; do shasum -a 256 _dist/aks-engine-$TAG-$tarball.tar.gz; done
+9ce3b35fc1395d4086cc47f86cc5b881936b00dbbba23ec40761215c943c3757  _dist/aks-engine-v0.33.3-darwin-amd64.tar.gz
```

Comment:
nit: the rest of the doc use `v0.32.3` as the example release so can we use it here too for consistency?

Reply:
done